README.md

```diff
@@ -131,6 +131,9 @@ The code is licensed under the MIT license.
 - Christian Grigis, C4DT
 - Laurent Girod, SPRING Lab, EPFL
 
+External contributors:
+- Benjamin Erhart, https://github.com/tladesignz
```

Comment:
@tladesignz - if you'd like another mention here, please say so.

Reply:
This is fine, no worries.